sdk/keyvault/azkeys/client.go

```diff
@@ -1065,6 +1072,7 @@ func (l *ListDeletedKeysOptions) toGenerated() *generated.KeyVaultClientGetDelet
 
 // ListDeletedKeys lists all versions of the specified key. The full key identifier and attributes are provided
```

Comment:
"specified key" sounds like there should be a key name/id parameter. If this function lists all the deleted keys in the vault, then I suggest something like the following:

```suggestion
// ListDeletedKeys lists all versions of the deleted keys. The full key identifier and attributes are provided
```

Reply:
Actually, this looks like it may have been a copy/paste from ListKeyVersions so perhaps this comment needs to be updated completely.